courseworks/week1/Tinho_WHU/hw1.md

```diff
@@ -0,0 +1,68 @@
+Q1 
+3.2M*8(cores)*4(SIMD)*2(fma)*4(avx2) = 0.8192 G FLOPS
+支持avx2 支持fma
+
+cat /proc/cpuinfo
+cpu MHz         : 3194.011
+cpu cores       : 8
+flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov pat pse36 clflush mmx fxsr sse sse2 ht syscall nx mmxext fxsr_opt pdpe1gb rdtscp lm constant_tsc rep_good nopl tsc_reliable nonstop
+_tsc cpuid extd_apicid pni pclmulqdq ssse3 fma cx16 sse4_1 sse4_2 movbe popcnt aes xsave avx f16c rdrand hypervisor lahf_lm cmp_legacy svm cr8_legacy abm sse4a misalignsse 3dnowprefetch osvw topoext perfctr_core
+ ssbd ibrs ibpb stibp vmmcall fsgsbase bmi1 avx2 smep bmi2 erms invpcid rdseed adx smap clflushopt clwb sha_ni xsaveopt xsavec xgetbv1 xsaves clzero xsaveerptr arat npt nrip_save tsc_scale vmcb_clean flushbyasid
+ decodeassists pausefilter pfthreshold v_vmsave_vmload umip vaes vpclmulqdq rdpid fsrm
+
+
+
+Q2
+cpu time 1s/3.2g = 0.3ns
+latency = 10cm/(3*10^8m /s) = 0.33ns
```

Comment:
A message will go twice though the channel, so the latency should be two times of $l/c$.